### PR TITLE
Update for the 2.0 wrappers.

### PR DESCRIPTION
**Pull request by TorHou**
I adapted the OpenSwathWorkflow.xml to incorporate JJ's changes.

### CODE REVIEW

File: tools/openms/IDFilter.xml [hunk 1]
```diff
@@ -70,6 +70,17 @@
 #if $param_whitelist_by_seq_only:
   -whitelist:by_seq_only
 #end if
+
+#if $rep_param_whitelist_protein_accessions:
+-whitelist:protein_accessions
+  #for token in $rep_param_whitelist_protein_accessions:
+    #if " " in str(token):
+      "$token.param_whitelist_protein_accessions"
```
Comment (bgruening) [inline tools/openms/IDFilter.xml]: This should be the standard, no matter what. Even better "${var_name}"

File: tools/openms/IsobaricAnalyzer.xml [hunk 2]
```diff
@@ -224,7 +224,7 @@
       <option value="tmt6plex">tmt6plex</option>
     </param>
     <param name="param_in" type="data" format="mzml" optional="False" label="input raw/picked data file" help="(-in) "/>
-    <param name="param_id_pool" type="text" size="30" label="ID pool file to DocumentID's for all generated output files" help="(-id_pool) Disabled by default. (Set to 'main' to use /home/thouwaar/Downloads/openms-2.0-Linux-x86_64/share/OpenMS/IDPool/IDPool.txt)">
+    <param name="param_id_pool" type="text" size="30" label="ID pool file to DocumentID's for all generated output files" help="(-id_pool) Disabled by default. (Set to 'main' to use /home/houwaart/Projects/OpenMS2.0/bin_15_07_2015/share/OpenMS/IDPool/IDPool.txt)">
```
Comment (bgruening) [inline tools/openms/IsobaricAnalyzer.xml]: Not good, here is a PATH ...

File: tools/openms/OMSSAAdapter.xml [hunk 3]
```diff
@@ -59,8 +59,13 @@
 #if $param_v:
   -v $param_v
 #end if
-#if $param_e:
-  -e $param_e
+#if $param_enzyme:
+  -enzyme
+  #if " " in str($param_enzyme):
+    "$param_enzyme"
```
Comment (bgruening) [inline tools/openms/OMSSAAdapter.xml]: See above, we should get rid of the #if " " ...

File: tools/openms/TextExporter.xml [hunk 4]
```diff
@@ -36,6 +36,9 @@
 #if $param_feature_minimal:
```
Comment (bgruening) [inline tools/openms/TextExporter.xml]: Is this tool really needed, or should this be on our blacklist?

**Comment by bgruening** [comment]
Awesome @TorHou! Are these changes also included? https://github.com/bgruening/galaxytools/pull/122

File: tools/openms/IDFilter.xml (hunk 1 above)
Comment (TorHou) [inline tools/openms/IDFilter.xml]: you mean 
"$token.param_whitelist_protein_accessions"
and not:
$token.param_whitelist_protein_accessions

ever?

File: tools/openms/IsobaricAnalyzer.xml (hunk 2 above)
Comment (TorHou) [inline tools/openms/IsobaricAnalyzer.xml]: I guess.. I have to check the script

File: tools/openms/IDFilter.xml (hunk 1 above)
Comment (bgruening) [inline tools/openms/IDFilter.xml]: Yes.

File: tools/openms/IsobaricAnalyzer.xml (hunk 2 above)
Comment (TorHou) [inline tools/openms/IsobaricAnalyzer.xml]: I blacklisted id_pool. If people need this, we have to find out what this does .. :)

**Comment by TorHou** [comment]
Yes #122 is in.

**Comment by TorHou** [comment]
I'm trying to converge with the latest CTD2Galaxy version ...

**Pull request merged**